_monitoring-your-cluster/pa/index.md

```diff
@@ -8,57 +8,100 @@ redirect_from:
+Then remount the file system:
+
+```bash
+mount -o remount /dev/shm
+```
 
-To start the performance analyzer root cause analysis (RCA) agent on a tarball installation, run the following command:
+### Security 
+
+Performance analyzer supports encryption in transit for requests. It currently does *not* support client or server authentication for requests. To enable encryption in transit, edit `performance-analyzer.properties` in your `$OPENSEARCH_HOME` directory.
+
+```bash
+vi $OPENSEARCH_HOME/config/opensearch-performance-analyzer/performance-analyzer.properties
+```
+
+Change the following lines to configure encryption in transit. Note that `certificate-file-path` must be a certificate for the server, not a root certificate authority (CA).
+
+````bash
+https-enabled = true
+
+#Setup the correct path for certificates
+certificate-file-path = specify_path
+
+private-key-file-path = specify_path
+````
+
+## Install Performance Analyzer 
+
+The Performance Analyzer plugin is included in the installation for [Docker]({{site.url}}{{site.baseurl}}/opensearch/install/docker/) and [tarball]({{site.url}}{{site.baseurl}}/opensearch/install/tar/). 
+
+If you need to install the Performance Analyzer plugin manually, download the plugin from [Maven](https://search.maven.org/search?q=org.opensearch.plugin) and install the plugin using the standard [plugins install]({{site.url}}{{site.baseurl}}/opensearch/install/plugins/) process. Performance analyzer will run on each node in a cluster.
+
+To start the Performance Analyzer  root cause analysis (RCA) agent on a tarball installation, run the following command:
       
 ````bash
 OPENSEARCH_HOME=~/opensearch-2.2.1 OPENSEARCH_JAVA_HOME=~/opensearch-2.2.1/jdk OPENSEARCH_PATH_CONF=~/opensearch-2.2.1/bin ./performance-analyzer-agent-cli
 ````
 
-The following command enables the performance analyzer plugin and performance analyzer RCA agent:
+The following command enables the Performance Analyzer plugin and Performance Analyzer  RCA agent:
 
 ````bash
 curl -XPOST localhost:9200/_plugins/_performanceanalyzer/cluster/config -H 'Content-Type: application/json' -d '{"enabled": true}'
 ````
 
-To shut down the performance analyzer RCA agent, run the following command:
+## Disable Performance Analyzer
 
-````bash
-kill $(ps aux | grep -i 'PerformanceAnalyzerApp' | grep -v grep | awk '{print $2}')
-````
+If you prefer to save memory and run your local instance of OpenSearch with the Performance Analyzer plugin enabled, use the following steps.
 
-To disable the performance analyzer plugin, run the following command:
+1. Shut down the Performance Analyzer  RCA agent by running the following command:
 
-````bash
-curl -XPOST localhost:9200/_plugins/_performanceanalyzer/cluster/config -H 'Content-Type: application/json' -d '{"enabled": false}'
-````
+  ```bash
+  kill $(ps aux | grep -i 'PerformanceAnalyzerApp' | grep -v grep | awk '{print $2}')
```

Comment:
Instead of Shell Command, can we use the following API:
```
curl -XPOST localhost:9200/_plugins/_performanceanalyzer/rca/cluster/config -H 'Content-Type: application/json' -d '{"enabled": false}'
```
and mention waiting for 5s for the disable to take effect before executing the next step.